pkg/services/publicdashboards/database/database.go

```diff
@@ -195,3 +195,21 @@ func (d *PublicDashboardStoreImpl) PublicDashboardEnabled(ctx context.Context, d
 
 	return hasPublicDashboard, err
 }
+
+func (d *PublicDashboardStoreImpl) PublicDashboardAccessTokenExists(ctx context.Context, accessToken string) (bool, error) {
```

Comment:
Do we need to use the prefix `PublicDashboard` in this methods after the refactor?